fastparquet/test/test_converted_types.py

```diff
@@ -40,7 +40,7 @@ def test_date():
     assert (convert(pd.Series([days]), schema)[0] ==
             pd.to_datetime([datetime.date(2004, 11, 3)]))
 
-
+@pytest.mark.skip(reason="fails")
```

Comment:
I do not know how this test passed.  The rhs (`np.array([731888], dtype='timedelta64[ms]')`) is a one-element array, so at the very least it should read `np.array([731888], dtype='timedelta64[ms]')[0]`, but even then the datatypes do not match

Reply:
`convert()` creates a datetime value (because of `converted_types=`). The single-element comparison is OK with numpy, although it raises a depricationwarning.
Does it fail for you?

Reply:
Yes, this test fails on Python2.7 windows.  I will remove the `skip` for this test, and add it as a separate issue.

Reply:
It seems that this requires more investigation. I wonder if it is a 32/64-bit integer problem. Note that you can use `skipif` to only avoid the test specifically on win32, e.g., something like https://docs.pytest.org/en/latest/skipping.html#id1